index.html

```diff
@@ -1,168 +1,139 @@
-
-          <div class="recent-work-item article-cnt">
-            <article>
-              <div class="article-item">
-                <div class="article-item-1">
-                  <img src="github-logo-octocat-800x665.png" alt="" width="200" height="200">
-                </div>
-                <div class="articel-item-2">
-                  <a href="#">Github</a>
-                </div>
-              </div>
-            </article>
-
-            <article>
-              <div class="article-item">
-                <div class="article-item-1"><img src="codepen-logo-600x300.png" alt="" width="200" height="200">
+    <div class="container">
+        <header>
+            <div id="title">
+                <h1><span class="thin">Michi</span>|<span class="bold">Berger</span></h1>
+                <p>web developement student at neuefische gmbh</p>
+            </div>
+            <nav>
+                <ul>
+                    <li><a class="btn" href="#contact">Contact</a></li>
+                    <li><a class="btn" href="#skills">Skills</a></li>
+                    <li><a class="btn" href="#works">Works</a></li>
+                    <li><a class="btn" href="#about-me">Biography</a></li>
+                </ul>
+            </nav>
+        </header>
+        <main>
+            <section  id="skills">
+                <figure>
+                    <img src="https://source.unsplash.com/random/600x400" alt="" width="300" height="auto">
+                    <figcaption class="thin">A random Picture from unsplash.com</figcaption>
+                </figure>
+                <div class="skills-ctn">
+                    <div class="skills-item">
+                        <h2>Skills</h2>
+                        <ul>
+                            <li class="btn">html</li>
+                            <li class="btn">css</li>
+                            <li class="btn">javascript</li>
+                            <li class="btn">github</li>
+                            <li class="btn">codepen</li>
+                            <li class="btn">wordpress</li>
+                        </ul>
+                    </div>
+                    <div class="skills-item">
+                        <h2>Interests</h2>
+                        <ul>
+                            <li class="btn">coding</li>
+                            <li class="btn">surfing</li>
+                            <li class="btn">football</li>
+                            <li class="btn">travelling</li>
+                        </ul>
+                    </div>
                 </div>
-
-                <div class="articel-item-2">
-                  <a href="#">Codepen</a>
+            </section>
+            <section id="contact">
+                <h2 class="contact-item">Contact</h2>
+                <address class="contact-item">
+                    <div class="address-item btn">
+                        <i class="fas fa-phone"></i>
```

Comment:
For what are you using the <i  as there is no content in it. If you do not add any i think you can take it  out and add the class to the <div above. 
In general i am wondering if there are too many <div s - at least i do not have so many, so might be that i am wrong in the end. for instant here, would <address  cover the section already so that you could just take it out?
so far it looks super structured, im impressed🤩

Reply:
hey jelena! thx a lot for your comments ;) I´ll check it again tomorrow...for sure there are to much divs :D
the `i` tag is a special tag for an icon from fontawesome...so it will be empty in my case!
thx again for the hints ;)